static/examples/change-material/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="">
+
+<head>
+  <meta charset="utf-8" />
+  <title>Getting Started with Vertex</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@vertexvis/viewer@0.19.x/dist/viewer/viewer.css" />
```

Comment:
Moving to `jsdelivr` here due to some performance issues we've been seeing with `unpkg`. This seems to only be an issue in the dev portal, so I'm open to revisiting this change if it feels worth the time